code/modules/projectiles/guns/energy/stun.dm

```diff
@@ -21,6 +28,7 @@
 	icon_state = "advtaser"
 	ammo_type = list(/obj/item/ammo_casing/energy/electrode, /obj/item/ammo_casing/energy/disabler)
 	origin_tech = null
+	fire_delay = 5

```

Comment:
Shouldn't this be 10 and not 5? Essentially keeping it on taser, from the start of the round, would be advantageous to shave off that 0.5 a second delay as opposed to switching it from stun to disable then back to stun.